edx_dl/parsing.py

```diff
@@ -376,7 +376,7 @@ def _make_url(section_soup):  # FIXME: Extract from here and test
 
         def _get_section_name(section_soup):  # FIXME: Extract from here and test
             try:
-                return section_soup.div.h3.string
+                return section_soup.find_all("h3", class_="section-title")[0].string
```

Comment:
@anmolsahoo25, is this still the case? Apparently, if I understood things correctly, there is some merge conflict with the recent changes that I pushed to the master branch...

Reply:
@rbrito , Yes. I am working on it. Will try to fix it ASAP. Pretty new to contributing on Github, so I'm just getting used to it.